project_app/components/Footer.js

```diff
@@ -7,17 +7,38 @@ import {
   ProductContainer,
 } from "./styled";
 import Image from "next/image";
+import Link from "next/link";
 
 export default function Footer() {
-  const titles = ["About Us", "FAQ", "Contact Us", "Give Feedback"];
-
+  const titles = [
+    {
+      name: "About",
+      link: "/About",
```

Comment:
links should be kebabcase
for example:
```suggestion
      link: "/about",
```

Reply:
That breaks the link as `export default About` , I'm I missing something